lib/rules/sotd/charter.js

```diff
@@ -7,14 +7,26 @@ const self = {
 
 exports.name = self.name;
 
+const util = require('../../util');
+
 const charterText =
     /The disclosure obligations of the Participants of this group are described in the charter\./;
 
 exports.check = async function (sr, done) {
     const sotd = sr.getSotDSection();
     if (sotd) {
-        const deliverId = await sr.getDelivererIDs();
-        if (!deliverId.length) {
+        const deliverIds = await sr.getDelivererIDs();
+
+        // Skip check if the document is only published by TAG or AB
+        const TagID = util.TAG.id;
+        const AbID = util.AB.id;
+        if (
+            deliverIds.length === 1 &&
+            (deliverIds[0] === TagID || deliverIds[0] === AbID)
+        )
+            return done();
+
+        if (!deliverIds.length) {
```

Comment:
That `if` should be the first one (before even checking if it's the TAG or the AB.